.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
What? I thought you wanted tabs? They were used everywhere. I only added an exception for `*.yml` because it's best-practice to use spaces there.